package.json

```diff
@@ -14,16 +14,19 @@
     "test": "npm run lint && npm run cover"
   },
   "dependencies": {
-    "async": "^0.9.0"
+    "async": "^0.9.0",
+    "react": "^0.12.0"
   },
   "devDependencies": {
     "chai": "^2.0",
     "coveralls": "^2.11.1",
     "eslint": "^0.15.0",
     "istanbul": "^0.3.5",
+    "jsdom": "^3.0",
     "mocha": "^2.0",
     "mockery": "^1.0",
     "precommit-hook": "^1.0",
+    "sinon": "^1.12.2",
```

Comment:
This is a dev dep so we can be more strict, figured it would be better to keep it as up to date as possible since those updates all help with performance and fix critical issues